### PR TITLE
Set steam branch based on whether STEAM_BRANCH is set

### PR DESCRIPTION
**Pull request by arron1993**
Been having trouble with my server not downloading the CDLC's.

Seems to be because -beta flag was not being applied as I only set the STEAM_BRANCH and not the password.

### CODE REVIEW

File: launch.py [hunk 1]
```diff
@@ -25,10 +25,11 @@ def env_defined(key):
 # Install Arma
 
 steamcmd = ["/steamcmd/steamcmd.sh"]
-steamcmd.extend(["+login", os.environ["STEAM_USER"], os.environ["STEAM_PASSWORD"]])
+steamcmd.extend(["+login", os.environ["STEAM_USER"],
+                os.environ["STEAM_PASSWORD"]])
```
Comment (BrettMayson) [inline launch.py:29]: If you revert these changes here it'll be good to merge in

**Review by BrettMayson — CHANGES_REQUESTED** [review]

File: launch.py (hunk 1 above)
Comment (arron1993) [reply launch.py]: Should be done
